service-workers/service-worker/resources/extendable-event-async-waituntil.js

```diff
@@ -45,6 +45,16 @@ self.addEventListener('message', function(event) {
       case 'script-extendable-event':
         self.dispatchEvent(new ExtendableEvent('nontrustedevent'));
         break;
+      case 'no-current-extension-different-microtask-after-dispatch-flag-unset':
```

Comment:
It seems more like 'current-extension-different-microtask-after-dispatch-flag-unset' as `event.waitUntil(waitPromise)` is called in the event handler itself?

Reply:
Good point, I just changed it.